inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/eltwise_b_fs_yx_fsv16.cl

```diff
@@ -58,22 +66,37 @@ KERNEL(eltwise_b_fs_yx_fsv16)(INPUTS_DECLS
 
     DO_ELTWISE
 
+#if HAS_FUSED_OPS
+    FUSED_OPS;
+#if BLOCK_SIZE != 1
+    OUTPUT_TYPE_BLOCK out = TO_OUTPUT_TYPE_BLOCK(FUSED_OPS_RESULT);
```

Comment:
`MAKE_VECTOR_TYPE` can properly handle vector size of 1, expanding to simple original type, which makes this `#if BLOCK_SIZE != 1` pointless.